src/collective/ifttt/tests/test_iftttaction.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from collective.ifttt.actions.ifttt import IftttEditFormView
+from collective.ifttt.actions.ifttt import IftttTriggerAction
+from plone.app.testing.bbb import PloneTestCase
+from plone.contentrules.rule.interfaces import IExecutable
+from plone.contentrules.rule.interfaces import IRuleAction
+from Products.statusmessages import STATUSMESSAGEKEY
+from Products.statusmessages.adapter import _decodeCookieValue
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+from zope.interface import implementer
+from zope.interface import Interface
+
+
+@implementer(Interface)
+class DummyEvent(object):
+    pass
+
+
+class IftttTests(PloneTestCase):
```

Comment:
@Shriyanshagro PloneTestCase should no longer be used. Check https://github.com/collective/collective.ifttt/blob/master/src/collective/ifttt/tests/test_setup.py how to inherit from unittest.TestCase and set layer attribute on the class to define integration test layer.